docs/DeveloperGuide.md

```diff
@@ -444,11 +458,11 @@ Priorities: MVP (must have), 2 (nice to have), 3 (unlikely to have)
       Use case resumes from Step 2.
 
 * **1b**. No contacts found matching the entered keyword.
-    * **1b1**. TutorEase displays a message: "No contacts found with the given keyword(s)."  
+    * **1b1**. TutorEase displays an error message, telling no contacts have been found with the given keyword.  
```

Comment:
maybe we can use
TutorEase displays an error message, saying no contacts have been found with the given keyword.

Reply:
maybe use "showing" instead of "saying"?

Reply:
Fixed!